libr/io/p/io_debug.c

```diff
@@ -1,5 +1,6 @@
 /* radare - LGPL - Copyright 2007-2019 - pancake */
 
+#define _GNU_SOURCE
```

Comment:
I'm not sure we want to depend on this. @trufae ? This could break on some non-gnu platforms? I'm not sure what they are, but i think there are some we want to support.

Reply:
i assume this is needed for `execvpe` which is not available on bsd or mac for example. isnt execve enough?